### ISSUE TITLE
As a ..., I want to ..., So I can ...

### ISSUE THREAD

**Issue by pautva** [micronutrientsupport/micronutrient-support-tool#500]
# Isue Status

![Page Status](https://user-images.githubusercontent.com/72566636/119643881-330c4e80-be14-11eb-918c-b8d45a266c58.png)

# Summary
An overview card for scenarios page; same as on the Baseline page

[Figma prototype](https://www.figma.com/proto/swhv1D7WScZwDE5KKYkkaE/BMGF-Maps?node-id=1767%3A2426&viewport=-4666%2C-4960%2C0.269345760345459&scaling=contain&page-id=0%3A1&hide-ui=1)

# Wireframe
![image](https://user-images.githubusercontent.com/72566636/120494668-ea283d00-c3b3-11eb-8cb4-9eeb2c1ca7ba.png)

# Location

Diet Data > Explore Simple dietary change scenarios

Part of scenarios page:
#499 

# Acceptance criteria

- [ ] Summary card is visible on Scenarios page
- [ ] It is the same as the one Baseline page
- [ ] The card can be expanded and collapsed

**Comment by jon571** [comment]
The "Baseline assumption" doesn't currently show on the baseline page (does in prototype, not in app).  Should it, and should it be shown here?

**Comment by pautva** [comment]
No, I think we don't need it. We have it shown on the "Projections" page, where it's more relevant. I'll replace the mockup screenshot.

**Comment by jon571** [comment]
I believe this task is complete then :-)

**Comment by Fuhji** [comment]
ready to close @bgsandan ?
